lib/elixir_internal_certificate_web/views/api/v1/keyword_upload_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.KeywordUploadView do
+  use JSONAPI.View, type: "keyword_upload"
```

Comment:
If we would like to return a response for this API, it will be the user_search type, not `keyword_upload`